indent/verilog_systemverilog.vim

```diff
@@ -220,21 +227,36 @@ function! s:GetContextIndent()
             \ s:curr_line =~ '^\s*' . s:vlog_open_statement &&
             \ s:curr_line !~ '^\s*/\*' &&
             \ s:curr_line !~ s:vlog_comment && !s:IsComment(v:lnum)
-        let l:offset += s:offset
-        call s:Verbose("Increasing indent for an open statment.")
+        let l:extra_offset += s:offset
+        call s:Verbose("Increasing indent for an open statement.")
+        let l:look_for_open_assign = 1
       endif
       let l:look_for_open_statement = 0
     endif
 
+    if l:look_for_open_assign == 1
+      " Search for assignments (=, <=) that don't end in ";"
+      if l:line =~ '[^=!]\zs=\ze[^=]\?' && l:line !~ ';\s*$'
```

Comment:
I was matching the original look around pattern, but now that I think about it that is not really required. I'll remove it.